README.md

```diff
@@ -1,4 +1,33 @@
-# ama-tool
+# ama-tool (Ask me Anything)
+
+[![GitHub version](https://badge.fury.io/gh/coding-bootcamps-eu%2Fama-tool.svg)](https://badge.fury.io/gh/coding-bootcamps-eu%2Fama-tool)
+[![npm version](https://badge.fury.io/js/npm.svg)](https://badge.fury.io/js/npm)
+[![ama-tool](https://img.shields.io/endpoint?url=https://dashboard.cypress.io/badge/simple/mikw99/dev&style=flat&logo=cypress)](https://dashboard.cypress.io/projects/mikw99/runs)
```

Comment:
Here is a message: No tests
Is this correct?

Reply:
is correct because the tests don't work atm. it's live updated with every test run